NxMBodyProblemCUDA/src/Parser/Parser.cs

```diff
@@ -0,0 +1,206 @@
+                                                                        semimajoraxis, 
+                                                                        null, 
+                                                                        double.Parse(splited[3], CultureInfo.InvariantCulture), 
+                                                                        double.Parse(splited[4], CultureInfo.InvariantCulture), 
+                                                                        double.Parse(splited[5], CultureInfo.InvariantCulture),
+                                                                        null, 
+                                                                        null, 
+                                                                        null, 
+                                                                        null, 
+                                                                        null, 
+                                                                        null, 
+                                                                        double.Parse(splited[7], CultureInfo.InvariantCulture));
+                astronomicalBodyList.Add(astronomicalBody);
+                c++;
+                //if (c == 30000)
+                //    break;
+            }
+
+            foreach(var a in astronomicalBodyList)
+            {
+                celestialBodyList.Add(a.CalculatePositionAndVelocity(2458200.5));
+            }
+
+            streamReader.Close();
+
+            Console.WriteLine("Readed " + astronomicalBodyList.Count.ToString() + " astronomical objects");
+        }
+
+        [STAThread]
+        public static void LoadData()
+        {
+            AstronomicalBody Earth = new AstronomicalBody("Earth", 0.01671123, 1.0, null, 0.0, -11.26064 / 180.0 * Math.PI,
+                114.20783 / 180.0 * Math.PI, null, 102.93768193 / 180.0 * Math.PI, null, 0.0, null, 5.97219e24,
+                2458236.784053135587);
+            AstronomicalBody Ceres = new AstronomicalBody("Ceres", .07553461024389638, 2.767046248500289, 2.558038488592984,
+                10.5935097971363 / 180 * Math.PI, 80.30991865594387 / 180 * Math.PI, 73.11534200131032 / 180 * Math.PI,
+                null, null, null, 352.2304611765882 / 180 * Math.PI, null, 62.6284 / Constants.G,
+                2458236.784053135587);
+            //CelestialBodyWithMass sun = new CelestialBodyWithMass("Sun",
+            //                                      new double[] { 1.953986474447199E+05, 9.744009844198005E+05, -1.620003654771077E+04 },
+            //                                      new double[] { -1.107391391838082E-02, 7.710256418221521E-03, 2.640571877471076E-04 },
+            //                                      Constants.Ms);
+            CelestialBodyWithMass sun = new CelestialBodyWithMass("Sun",
```

Comment:
Feature: Add new method for adding new celestial bodies with mass - as reading from file.